### DISCUSSION TITLE
multiple map leaders

### DISCUSSION THREAD

**Discussion by sourcevault** [NeoVintageous/NeoVintageous#882] [General]
• https://old.reddit.com/r/vim/wiki/the_leader_mechanism 
﹏﹏﹏﹏﹏﹏﹏﹏﹏﹏﹏﹏﹏﹏﹏﹏﹏﹏﹏﹏﹏﹏﹏
Vim makes it possible to formally define two leaders:
```
let mapleader = ","
let localmapleader = "\<Space>"
```
﹋﹋﹋﹋﹋﹋﹋﹋﹋﹋﹋﹋﹋﹋﹋﹋﹋﹋﹋﹋﹋﹋﹋
would it be possible to introduce `localmapleader` variable ?

my default `let mapleader=<space>` is getting too crowded.

would be nice to do a `let localmapleader = <tab>` too.

https://github.com/NeoVintageous/NeoVintageous/issues/308#issuecomment-369446848

For now, it would be easier to solve the above problem with multiple leader keys as the command strings gets into 4 char size.  it's also easier on the hand.

**Comment by gerardroche** [comment]
https://github.com/NeoVintageous/NeoVintageous/issues/883

**Comment by gerardroche** [comment]
It looks like this was already implemented. I think you might have a typo.

The default for `<LocalLeader>` is `<bslash>` (backslash).

Example:

```vim
let maplocalleader=<tab>
nnoremap <LocalLeader>/ :
```

There can be no spaces before or after `let maplocalleader=`.

**Reply by sourcevault** [reply]
There is no syntax highlighting for `<LocalLeader>`

![image](https://github.com/NeoVintageous/NeoVintageous/assets/20347647/31d6e8d0-64a4-45fb-bd10-6c766451ca14)
